README.md

```diff
@@ -24,6 +26,15 @@ To run the server, you need to create a MongoDb Atlas instance and connect your
 
 __Congrats!__ You're ready to run locally! Don't hesitate to reach out if you run into issues. 
 
+### Read `server/framework/doc.ts`
+
+To understand how to perform different CRUD operations on your MongoDB database, read `doc.ts`. You will use these functions to help you solve the `TODOs` for this prep.  
+
+## Complete the TODOs in `server/concepts/user.ts`
```

Comment:
The TODOs are in `routes.ts` and `user.ts` (possibly `post.ts` too if they want to edit that)